webpack.config.js

```diff
@@ -27,6 +27,7 @@ module.exports = (env) => ({
           {
             include: [
               path.resolve(__dirname, 'src/illustrativeIcons'),
+              // Component icons rely on having specific id's and classnames. These are used inside suomifi-ui-components library.
               path.resolve(__dirname, 'src/componentIcons')
```

Comment:
I'd separate these two since component icons rely on different classnames and should not include illustrative classnames to avoid potential conflicts when using custom styles.